src/app/components/MobileFooterNavigation/index.tsx

```diff
@@ -23,15 +25,27 @@ const StyledMobileFooterNavigation = styled.nav`
   right: 0;
   height: ${mobileFooterNavigationHeight};
   display: flex;
-  align-items: center;
-  justify-content: space-evenly;
+  align-items: stretch;
+  justify-content: space-between;
   box-shadow: ${({ theme }) =>
     `0px 0px ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-front-border', theme)}`};
   border-top: ${({ theme }) =>
     `solid ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-contrast', theme)}`};
   flex-direction: row;
 `
 
+const StyledNavLink = styled(NavLink)`
+  // Make items mostly-equal width
+  flex-grow: 1;
+  flex-basis: 0;
+  &:first-of-type {
```

Comment:
To make it 110% width compared to other items, due to text probably cutting out. But this only works on the extension width size. It would fail on tablet size I guess - not sure about when the mobile tabs are displayed exactly.

Reply:
> what was a reason to change only first item? mimic white-space: nowrap;?

Yeah but it already seems to fit into one line now, so can remove